dvc/repo/__init__.py

```diff
@@ -155,7 +157,8 @@ def _ignore(self):
 
     def check_dag(self, stages):
```

Comment:
can we actually change names a bit - stages -> new_stages or check_modified_dag?

Reply:
it looks like it's used in a few other places and should be updated?